src/App.js

```diff
@@ -25,7 +26,8 @@ function App() {
   };
 
   const handleAddMealBtn = () => {
-    const new_meal = { name: inputIngridientName, ingridients };
+    const lastId = mealsData[mealsData.length - 1].id;
+    const new_meal = { name: inputIngridientName, ingridients, id: lastId + 1 };
```

Comment:
Follow the same naming
```suggestion
    const newMeal = { name: inputIngridientName, ingridients, id: lastId + 1 };
```

Reply:
ok